### ISSUE TITLE
Does the library manage instances?

### ISSUE THREAD

**Issue by robinclaes** [mrousavy/react-native-mmkv#449]
Hi, for our use case (having an encrypted storage per user), we are looking into this library.

We want to be able to access the instance from anywhere in our code.
We are wondering if we need to manage this instance ourself? (e.g. singleton pattern)

What will happen if we create a new instance with an existing id? For example:
```typescript
const instance = new MMKV({ id: 'user-storage' }); // a storage with this id already exists, will it open or override it?
```
Does the database get overwritten, or does it open the existing database?
In other words I guess - does the library manage the instances on the native side or do we have to do that ourselves?

**Comment by robinclaes** [comment]
After testing this I have found that the database does not get overwritten. Closing this.
